Carting/src/WebApi/Domain/ValueObjects/Currency.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Carting.WebApi.Domain.Common;
+using Carting.WebApi.Domain.Exceptions;
+
+namespace Carting.WebApi.Domain.ValueObjects;
```

Comment:
Good point, you're right.

One-field enumerations act as value objects because we can treat their instances interchangeably as long as their content is the same. 

I've changed it to an enum.